### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by pepdbm7**

### CODE REVIEW

File: staff/pep-delbano/app-template/about.html [hunk 1]
```diff
@@ -5,6 +5,84 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>About</title>
+    <style>
+        body {
+    background-color: rgb(114, 196, 92);
+    font-family: arial;
+  }
+
+  #container {
```
Comment (manuelbarzi) [inline staff/pep-delbano/app-template/about.html]: avoid using id selectors if not strictly required, use class selectors instead

File: staff/pep-delbano/cv/cv.html [hunk 2]
```diff
@@ -5,58 +5,62 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (manuelbarzi) [inline staff/pep-delbano/cv/cv.html]: rename  file to index.html

File: staff/pep-delbano/cv/styles.css [hunk 3]
```diff
@@ -17,93 +25,221 @@
...
+    -ms-transition: background-color 1500ms linear;
+    transition: background-color 1500ms linear;
 }
 
 .header {
-    grid-area: header;
-    width: 100%;
-    background-color: hsl(183, 69%, 87%);
-    padding: 20px;
+    max-width: 150px;;
```
Comment (diesttro) [inline staff/pep-delbano/cv/styles.css:43]: Double semicolon

File: staff/pep-delbano/cv/cv.html [hunk 4]
```diff
@@ -5,58 +5,62 @@
...
-    <aside class="rightside1">
-        <h2>Studies</h2> 
-        <p><b>Fisics degree:</b> Superior Politecnic School at Graz </p>
-        <p><b>Electrician degree: </b>UAB</p>
+        <header class="header"><h1 class="header_maintittle">CURRICULUM VITAE</h1></header>
+
+        <img class="image" src="https://4.bp.blogspot.com/-Hw_ZYszymn8/UUuXmfD9obI/AAAAAAAABbg/acWFBmhj1d4/s1600/TESLA.jpg" alt="Nikola Tesla portrait pic">
         
-    </aside><br>
+        <aside class="leftside">
```
Comment (manuelbarzi) [inline staff/pep-delbano/cv/cv.html]: aside is aimed to tangential content, but not for part of the main content, please RTFM https://www.google.es/url?sa=t&rct=j&q=&esrc=s&source=web&cd=2&cad=rja&uact=8&ved=2ahUKEwjTpqO5_uTdAhVEzqQKHQclDgwQFjABegQIBBAB&url=https%3A%2F%2Fdeveloper.mozilla.org%2Fen-US%2Fdocs%2FWeb%2FHTML%2FElement%2Faside&usg=AOvVaw38alzYAR_ziOyIpk3tYV3z

File: staff/pep-delbano/cv/cv.html [hunk 5]
```diff
@@ -5,58 +5,62 @@
...
-            <li>Neon Lamps </li>
-            <li>Alternating current</li>
-            <li>The Adams Power Plant Transformer House</li>
-        </ul>
-    </main>
-    
-    <footer class="neverBuilt">
-        <h2>Inventions that never got built</h2>
-        <ul>
-            <li><h3></h3></li>
```
Comment (b-rage) [inline staff/pep-delbano/cv/cv.html:52]: Why H3 empty here?

File: staff/pep-delbano/cv/styles.css [hunk 6]
```diff
@@ -17,93 +25,221 @@
...
     font-size: 35px;
-    font-family: "Lucida Sans Unicode", "Lucida Grande", sans-serif;
+    width: 90%;
+    font-family: 'Lobster', cursive;    
+    text-shadow: 1px 2px 3px white;
+    letter-spacing: 4px;
 }
 
-h2 {
+.div__head{
```
Comment (manuelbarzi) [inline staff/pep-delbano/cv/styles.css]: `div__head` is NOT semantic, may you wanted to mean something like `container__head` or similar? AVOID using `div` as a prefix for a BEM selector, it's agains the principle of building concepts (blocks, objects)

File: staff/pep-delbano/cv/styles.css [hunk 7]
```diff
@@ -17,93 +25,221 @@
...
+
+
+.leftside__email {
+    text-decoration: underline;
+    color: blue; 
+    display: inline-block;
+}
+
+
+.div__span--bold {
```
Comment (manuelbarzi) [inline staff/pep-delbano/cv/styles.css]: again here, avoid `div` as a prefix for a BEM class selector

File: staff/pep-delbano/cv/styles.css [hunk 8]
```diff
@@ -17,93 +25,221 @@
...
+    }
+    
+    .header {
+        font-size: 45px;
+        margin: 0 10px;
+        max-width: 300px;
+        width: 85%;
+    }
+
+    .div__head {
```
Comment (manuelbarzi) [inline staff/pep-delbano/cv/styles.css]: avoid `div_`, use a more semantic prefix

**Pull request merged**